schemas/context/consent.schema.json

```diff
@@ -0,0 +1,63 @@
+{
+	"meta:license": [
+		"Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+		"This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+		"you may not use this file except in compliance with the License. You may obtain a copy",
+		"of the License at https://creativecommons.org/licenses/by/4.0/"
+	],
+	"$id": "https://ns.adobe.com/xdm/context/consent",
+	"$schema": "http://json-schema.org/draft-06/schema#",
+	"title": "XDM Consent",
+	"type": "object",
+	"auditable": true,
+	"meta:extensible": true,
+	"meta:abstract": true,
+	"meta:extends": [
+		"https://ns.adobe.com/xdm/data/record"
+	],
+	"description": "An XDM Consent is a representation of the consent responses given by end users/consumers of a given brand to allow or not allow the Brand/Customers to use their personal data for analytics purposes like Direct Marketing, First Party personalization, cross device profile consolidation, advertsing, sending offers etc..",
+	"definitions": {
+		"consent": {
+			"properties": {
+				"Id": {
+					"title": "Id",
+					"description": "The unique identifier of the identity associated with this record.",
+					"type": "string"
+				},
+				"Ttl": {
+					"title": "Time to Live",
+					"description": "The date and time this entry is valid until.",
+					"type": "string",
+					"format": "date-time"
+				},
+				"ConsentTimestamp": {
+					"title": "Consent Timestamp",
+					"description": "The date and time the snapshot of consent assertions was taken.",
+					"type": "string",
+					"format": "date-time"
+				},
+				"ConsentSignal": {
+					"title": "Consent Signal",
+					"type": "string",
+					"description": "User consent signal for a purpose within a given consent stack.",
+					"default": "N",
```

Comment:
Can default have a value which is not present in enum?

Reply:
Since we do not have any current sample feed file, I am removing the default value as we are not sure if the signals would come as Y/N, 1/0 or TRUE/FALSE. Meta:Enum is just there for now to convey the idea. We can always evolve this overtime as we get more visibility into the data.